sonic_platform_base/sonic_xcvr/api/public/sff8436.py

```diff
@@ -68,21 +68,34 @@ def get_transceiver_info(self):
 
         return xcvr_info
 
-    def get_transceiver_bulk_status(self):
+    def get_transceiver_status(self):
         rx_los = self.get_rx_los()
         tx_fault = self.get_tx_fault()
         tx_disable = self.get_tx_disable()
         tx_disabled_channel = self.get_tx_disable_channel()
+        read_failed = rx_los is None or \
+                      tx_fault is None or \
+                      tx_disable is None or \
+                      tx_disabled_channel is None
+        if read_failed:
+            return None
+
+        trans_status = {
+            "rx_los": all(rx_los) if self.get_rx_los_support() else 'N/A',
```

Comment:
For CMIS, we are displaying rx_los, tx_fault and tx_disable per channel whereas for non-CMIS modules, we are combining the o/p in a single variable.
Can we make the o/p consistent for all transceiver types?

Reply:
made the format consistent for all types.